client/java/src/main/java/com/linecorp/centraldogma/client/updater/CentralDogmaBeanMethodHandler.java

```diff
@@ -18,24 +18,46 @@
 import static java.util.Objects.requireNonNull;
 
 import java.lang.reflect.Method;
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
 
 import com.linecorp.centraldogma.client.Watcher;
 
 import javassist.util.proxy.MethodHandler;
 
-class CentralDogmaBeanMethodHandler<T> implements MethodHandler {
-    private final Watcher<T> watcher;
-    private final T defaultValue;
+/**
+ * This class Handle method dispatch from original bean to the bean proxy.
+ *
+ * @param <T> type of bean will be handle of this class
+ */
+public final class CentralDogmaBeanMethodHandler<T> implements MethodHandler {
+    final Watcher<T> watcher;
+    final T defaultValue;
 
     CentralDogmaBeanMethodHandler(Watcher<T> watcher, T defaultValue) {
         this.watcher = watcher;
         this.defaultValue = requireNonNull(defaultValue, "defaultValue");
     }
 
+    // note that below delegate will be match from top to bottom
+    // so if your matcher may duplicate with another one, please consider the order carefully
+    private static final List<MethodDelegate> methodDelegators = ImmutableList.of(
+            new RevisionMethodDelegate(),
+            new CloseMethodDelegate()
+    );
+
+    private static final MethodDelegate defaultDelegator = new DefaultMethodDelegate();
```

Comment:
You could just move this into the last position of `methodDelegators`.

Reply:
i just want to prevent case that some people forgot to see the comment and put default one into the middle of delegator chain, which may cause it to ignore everything else after.
HDYT?

Reply:
Then how about doing this:

```java
methodDelegators = ... (
    ...
    // The last delegator must be DefaultMethodDelegator.
    new DefaultMethodDelegator());

for (d : methodDelegators) {
    ...
}

// Should never reach here.
throw new Error();
```

Reply:
sounds reasonable to me, applied